test/test_browser.py

```diff
@@ -5318,6 +5318,7 @@ def test_wasmfs_fetch_backend(self, args):
     create_file('subdir/backendfile2', 'file 2')
     self.btest_exit('wasmfs/wasmfs_fetch.c',
                     args=['-sWASMFS', '-pthread', '-sPROXY_TO_PTHREAD',
+                          '-sFORCE_FILESYSTEM', '-lfetchfs.js',
```

Comment:
How did this test work before with `fetchfs.js` included?

Reply:
It didn't use the JS library before, just the WASM backend code which was included either way.